website/docs/r/repository_deploy_key.html.markdown

```diff
@@ -21,12 +21,17 @@ Further documentation on GitHub repository deploy keys:
 ## Example Usage
 
 ```hcl
-# Add a deploy key
+# Generate an ssh key
+resource "tls_private_key" "example_repository_deploy_key" {
```

Comment:
I'm using assuming you're using the [hashicorp/tls](https://registry.terraform.io/providers/hashicorp/tls/latest/docs/resources/private_key) provider here. Can you provide a small comment noting the provider? 

Otherwise the changes look good!

Reply:
Done